test/services/GovSpendingServiceTest.scala

```diff
@@ -22,15 +22,26 @@ import uk.gov.hmrc.play.test.UnitSpec
 
 class GovSpendingServiceTest extends UnitSpec with MockitoSugar with ScalaFutures {
   "GovSpendingServiceTest" should {
-    "return a map containing the expected Welfare rate and have size 15" in {
+    "return a map containing the expected Welfare rate and have size 15 (year:2018)" in {
       val spending = GovSpendService.govSpending(2018)
       spending.get("Welfare").get should be (23.80)
       spending.size should be (15)
     }
-    "Throw MatchError for an invalid year" in {
+    "Throw MatchError for an invalid year (year:2018)" in {
+      assertThrows[MatchError] {
+        GovSpendService.govSpending(1900)
+      }
+    }
+    "return a map containing the expected Welfare rate and have size 15 (year:2019)" in {
+      val spending = GovSpendService.govSpending(2019)
+      spending.get("Welfare").get should be (23.50)
```

Comment:
Don't do `.get` here as it isn't needed you can just compare against a `Some`. It will give a more meaning full output should the test fail

```suggestion
      spending.get("Welfare") should be Some(23.50)
```